paws_collector.js

```diff
@@ -72,6 +73,15 @@ class PawsCollector extends AlAwsCollector {
         });
     }
 
+    // we can do whatever conditional handler wrapping here
+    static makeHandler(handlerFunc) {
+        if(process.env.DD_API_KEY || process.env.DD_KMS_API_KEY){
```

Comment:
we need to add these params to collector temaplates

Reply:
how are these env vars passed into a collector?

Reply:
I thought that we could add these when azcollect creates a collector.